frontend/src/domain/entities/mission/hooks/useGetFilteredMissionsQuery.ts

```diff
@@ -66,7 +61,8 @@ export const useGetFilteredMissionsQuery = (): {
       missionSource: listFilterValues[MissionFilterType.SOURCE],
       missionStatus: [listFilterValues[MissionFilterType.STATUS]],
       missionTypes: [listFilterValues[MissionFilterType.TYPE]],
-      seaFronts: filteredSeaFronts,
+      // seaFronts are filtered in memory
+      seaFronts: [],
```

Comment:
Une autre question : quand tu dis in memory, tu veux dire dans le LocalStorage via redux-persist ? Et si c'est le cas, in ne faut pas le récupérer à partir du store ici ?

Reply:
Ah je voulais dire que ce filtre est fait dans le frontend (avec les missions déjà chargées) plutôt que dans le back ; je passe un array vide car on ne demande plus au backend de filtrer, mais on récupère bien `filteredSeaFronts` depuis le store pour filtrer !